docs/latest/tutorial/tutorial-main-renderer.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Main and Renderer Process Communication'
+description: 'This guide will step you through the process of creating a barebones Hello World app in Electron, similar to electron/electron-quick-start.'
+slug: node-access
+hide_title: false
+---
+
+:::info Tutorial parts
+This is part 3 of the Electron tutorial. The other parts are:
+
+1. [Prerequisites]
+1. [Scaffolding]
+1. [Main and Renderer process communication][main-renderer]
+1. [Adding Features][features]
+1. [Application Distribution]
+1. [Code Signing]
+1. [Updating Your Application][updates]
+
+:::
+
+At the end of this part you will have an Electron application that uses
+a `preload` script to safely expose features from the main process
```

Comment:
Open to feedback here, but it might make sense to not code fence `preload` during normal usage because we use the term a lot.

```suggestion
a preload script to safely expose features from the main process
```